src/main/pages/user-management/permissions.adoc

```diff
@@ -196,9 +200,14 @@ Sample `body` to grant user `userID` permissions to a workspace `workspaceID`:
 }
 ----
 
+The `instanceId` parameter corresponds to the ID of the resource that retrieves the permission for all users. The `userId` parameter corresponds to the ID of the user that has been granted certain permissions.
+
 [id="sharing-permissions"]
-== Sharing Permissions
+== Sharing permissions
+
+A user with `setPermissions` privileges can share a workspace and grant `read`, `use`, `run`, `configure`, or `setPermissions` privileges to other users.
 
-A user with `setPermissions` privileges can share a workspace, i.e. grant other users `read, use, run, configure or setPermissions` privileges.
+To share workspace permissions:
 
-Select a workspace in User Dashboard, navigate to `Share` tab and enter emails of users to share this workspace with (use comma or space as separator if there are multiple emails).
+* Select a workspace in the user dashboard, navigate to the *Share* tab and enter emails of users. Use commas or space as separators:w
```

Comment:
Typo with `:w` here. Editing in Vim? :)

Reply:
yes I was.....will start using Atom

Reply:
Typos happen, no matter what editor you use. Don't feel you need to start using a diff. editor because of this.